api/src/pcapi/core/offers/exceptions.py

```diff
@@ -174,8 +174,10 @@ class ExtraDataValueNotAllowed(OfferCreationBaseException):
 
 class OfferAlreadyExists(OfferCreationBaseException):
     def __init__(self, field: str) -> None:
-        super().__init__(field, f"Une offre avec cet {field} existe déjà")
-
+        if field == "ean":
+            super().__init__(field, f"Une offre avec cet {field.upper()} existe déjà. Vous pouvez la retrouver dans l’onglet Offres.")
+        else:
+            super().__init__(field, f"Une offre avec cet {field} existe déjà")
```

Comment:
Puisque cette exception n'est levée que pour les champs "ean" et "isbn", je pense qu'on pourrait avoir le même comportement (le nouveau) dans tous les cas en supprimant la condition : 

```suggestion
    super().__init__(field, f"Une offre avec cet {field.upper()} existe déjà. Vous pouvez la retrouver dans l’onglet Offres.")
```

Reply:
Maintenant tu valide cette PR, stp?